cgi/product_multilingual.pl

```diff
@@ -1402,6 +1319,114 @@ ($product_ref, $field, $language)
 
 	$template_data_ref_display->{nutriments} = \@nutriments;
 
+	# Display 2 checkbox to indicate the nutrition values present on the product
+
+	if (not defined $product_ref->{nutrition_data}) {
+		# by default, display the nutrition data entry column for the product as sold
+
+		$product_ref->{nutrition_data} = "on";
+	}
+
+	if (not defined $product_ref->{nutrition_data_prepared}) {
+		# by default, do not display the nutrition data entry column for the prepared product
```

Comment:
we could try to be smart on dehydrated things

Reply:
I added something if we have a category that should have nutrition data